priiloader/Makefile

```diff
@@ -137,28 +156,54 @@ else
 	@echo "#define GIT_REV_STR \"0x$(shell git describe --always --long --dirty)\"" >> ../Shared/gitrev.h
 	@echo "Using 0x$(shell git describe --always --long) as revision, \"0x$(shell git describe --always --long --dirty)\" as revision string"
 endif
-#build
-	@[ -d $@ ] || mkdir -p $@
+
+#test if the build dir exists, make it if its not
+	@test -d $(BUILD) || mkdir $(BUILD)
+
+loader: $(DATA)/loader.bin
+$(DATA)/loader.bin: $(LDRFILES)
+	@echo -----------------------------------------------
+	@echo compiling the loader...
+	@echo -----------------------------------------------
+	@echo 
+#test if the build dir exists, make it if its not
+	@test -d $(BUILD) || mkdir $(BUILD)
+#the generated ASM by the following command has no purpose but for debugging needs.
+	@echo generating loader.s
+	@$(CC) --static -iquote include -I $(BUILD) -I$(LIBOGC_INC) -S loader\loader.c -o $(BUILD)\loader.s

```

Comment:
All the backslashes in the paths (Line 173 to line 178) need to be replaced with forward slashes, otherwise this doesn't compile on Linux.

Reply:
should be fixed now